test/name_mapper.cpp

```diff
@@ -61,7 +79,22 @@ class CapturedStderr
   operator std::string() const { return buffer.str(); }
 };
 
+#if _WIN32
+const std::string ZERO_FOUR_NAME_CONFLICT_MSG =
+    "Path collision: 'C:\\data\\zero_four_2021-10.zim' and"
+    " 'C:\\data\\zero_four_2021-11.zim' can't share the same URL path 'zero_four'."
+    " Therefore, only 'C:\\data\\zero_four_2021-10.zim' will be served.\n";
+
+const std::string ZERO_SIX_NAME_CONFLICT_MSG =
+    "Path collision: 'C:\\data\\zërô + SIX.zim' and "
+    "'C:\\data\\zero_plus_six.zim' can't share the same URL path 'zero_plus_six'."
+    " Therefore, only 'C:\\data\\zërô + SIX.zim' will be served.\n";
 
+const std::string ZERO_SEVEN_NAME_CONFLICT_MSG =
+    "Path collision: 'C:\\data\\subdir\\zero_seven.zim' and"
+    " 'C:\\data\\zero_seven.zim' can't share the same URL path 'zero_seven'."
+    " Therefore, only 'C:\\data\\subdir\\zero_seven.zim' will be served.\n";
+#else
```

Comment:
This (and the respective change regarding the input data for this test) is ugly, but I don't see how it can be improved in a nice way.

Reply:
I agree with you.